### DISCUSSION TITLE
Open Source Projects

### DISCUSSION THREAD

**Discussion by lnginyard** [community/community#28677] [Other Feature Feedback, Questions, & Ideas] [answered]
I've wanted to contribute to the open source for quite some time now because I think that it's a better way for me to socialize with developers of like minds and also make my virtual appearance more appeasable to prospective employers and future collaborators. Only thing is, I'm unsure of where to find a good open-source project to contribute to that doesn't require senior-level development. 

I'm here to get some suggestions from people who contribute to open-source projects or have been searching for projects to contribute to and have come across some good ones that are more for entry-level to mid-level developers. Hoping to hear a few good suggestions. Thanks!

**Comment by BergerAPI** [comment]
You can always look out for repositories that are starting through, and still have many open to-dos. On the Bun-Discord for example, you have a showcase channel, where people put their projects in and are seeking contributions. Other than that you can always look out for typos in repositories.

**Reply by lnginyard** [reply]
Thanks @BergerAPI for your comment! I didn't think that making a small edit such as a typo contributes to open-source projects. I mean, it makes sense now that you've mentioned it but didn't think of it in that way before. I see typos in open-source projects all the time and just mention it to myself, but now when I see them, I will edit and fix them to contribute.

**Comment by airtower-luna** [comment]
I suggest looking at software you use & like, and see if you can help out. Maybe there's a little bug you could fix, a test that'd make sense to add, or a documentation section that could use an extra example? Maybe there's even an improvement you'd like to see?

That way you don't have to get into something completely new, and it does wonders for confidence to use a tool and know you contributed to it. :smiley_cat:

Oh, and get into the habit of reporting bugs you encounter when you can. Reproducing bugs and writing good bug reports are extremely useful skills, and who knows, maybe you can fix one here and there.

**Reply by lnginyard** [reply]
This is great information you've given here! I know that looking for bugs and fixing them are the most time-consuming tasks in developing so, reporting them alone will definitely contribute in a huge way. Thanks for pointing that out.. I once caught a bug in an app I used but I opened up developer tools and made the change just to see it for myself but didn't think of coming to GitHub to report it but now since I do have an urge to contribute, I'll make sure I come here and report and/or fix the bug.

**Comment by melvincwng** [comment]
@lnginyard Hi Lorenzo, for a beginner friendly open source project, you can consider this: https://github.com/Your-First-Open-Source-Project

All the best in your open source contributions on GitHub 😊

**Reply by lnginyard** [reply]
Oh wow, thank you! This is spot on and all that I need to get started on contributing to my first open-source project! 🙌🏾👏🏾

**Comment by dogweather** [comment, marked as answer]
Many repos use the Issue tag `good first issue`. When they do, you'll see a note about "issues needing help" in an organization's repo list:

<img width="293" alt="Screen Shot 2022-08-15 at 5 19 38 PM" src="https://user-images.githubusercontent.com/150670/184738312-7a1f2f45-1461-4fd1-b3c0-f0fce583af39.png">

**Reply by lnginyard** [reply]
Thank you! I'll look out for that tag.

**Reply by dogweather** [reply]
> Thank you! I'll look out for that tag.

Great. For example, here's the filtered list from my open source repo: 

https://github.com/public-law/open-gov-crawlers/labels/good%20first%20issue
